test/e2e/service.go

```diff
@@ -455,3 +447,10 @@ func addEndpointPodOrFail(c *client.Client, ns, name string, labels map[string]s
 	_, err := c.Pods(ns).Create(pod)
 	Expect(err).NotTo(HaveOccurred())
 }
+
+// dateStamp returns the current time as a string "YYYY-MM-DDTHHMMSS"
+// Handy for unique names across test runs
+func dateStamp() string {
```

Comment:
wondering if instead we should have a function in test/e2e/util.go which generates a namespace name (perhaps parameterized on test name etc as well)?